src/serac/physics/CMakeLists.txt

```diff
@@ -37,15 +37,9 @@ set(physics_headers
     )
 
 set(physics_dependencies
-    serac_infrastructure
     serac_coefficients
-    serac_functional
     serac_physics_integrators
```

Comment:
the fact that some have `serac_physics_whatever` and others don't is inconsistent here...